Mathlib/Data/Vector.lean

```diff
@@ -258,4 +264,10 @@ theorem toList_take {n m : ℕ} (v : Vector α m) : toList (take n v) = List.tak
   rfl
 #align vector.to_list_take Vector.toList_take
 
+instance {α : Type u} {n : Nat} : GetElem (Vector α n) (Fin n) α (fun _ _ => True) where
+  getElem := fun x i _ => get x i
```

Comment:
This conflicts with another instance in core that has a different outparam.  I'd just put the same instance here like we do for arrays, i.e., only the `Nat` one.  https://github.com/leanprover/lean4/blob/bff612e59e4860bef5180db08356ab5f256bea86/src/Init/Data/Fin/Basic.lean#L120